src/main/frontend/src/code-quarkus/api/quarkus-project-utils.ts

```diff
@@ -88,8 +88,7 @@ export function resolveInitialProject(extensions: ExtensionEntry[]) {
 }
 
 const defaultCleanHistory = () => {
-  console.log(`remove query from url: ${window.location.search}`);
-  window.history.replaceState({}, document.title, window.location.href.replace(window.location.search, ''));
```

Comment:
This is going to break the "push to GitHub" and the "share feature":
ex: https://code.quarkus.io/?g=org.acme&a=code-with-quarkus&v=1.0.0-SNAPSHOT&b=MAVEN&s=YjV.pV1.49J&cn=code.quarkus.io

Instead of changing this behavior, I would extract the param you are looking for before the call to `parseProjectInQuery` and then pass it to the `ExtensionsPicker` using a prop..

Cleaning the query after reading it is important as we are not keeping it in sync with the search afterward.

Reply:
In order to not break those features, I reimplemented these lines. Another thing, I put the param that I need in a prop, and I set this param in the filter in `ExtensionsPicker`.